tst/CartesianClosedOpposite.tst

```diff
@@ -0,0 +1,1923 @@
+##################################################################################
+# Test the correspondences between the cartesian closed structure of the category
+# of finite sets and the cocartesian coclosed structure of its opposite.
+##################################################################################
+
+gap> START_TEST("CartesianClosedOpposite.tst");
+
+# LoadPackage("FinSets");
```

Comment:
```suggestion
gap> LoadPackage( "FinSetsForCAP", false );
true
```

This way, `FinSetsForCAP` will be loaded when calling the `tst` file directly.

Reply:
Thank you, I have no idea how that didn't occur to me. I'll also add it in Rows.